view/templates/field_input.tpl

```diff
@@ -1,8 +1,8 @@
 	
-	<div class='field input' id='wrapper_{{$field.0}}'>
-		<label for='id_{{$field.0}}'>{{$field.1}}</label>
-		<input{{if $field.6 eq 'email'}} type='email'{{elseif $field.6 eq 'url'}} type='url'{{elseif $field.6 eq 'int'}} type='tel'{{else}} type="text"{{/if}} name='{{$field.0}}' id='id_{{$field.0}}' value="{{$field.2 nofilter}}"{{if $field.4 eq 'required'}} required{{/if}}{{if $field.5 eq "autofocus"}} autofocus{{elseif $field.5}} {{$field.5 nofilter}}{{/if}} aria-describedby='{{$field.0}}_tip'>
+	<div class="field input" id="wrapper_{{$field.0}}">
+		<label for="id_{{$field.0}}">{{$field.1}}</label>
+		<input{{if $field.6 eq 'email'}} type="email"{{elseif $field.6 eq 'url'}} type="url"{{elseif $field.6 eq 'int'}} type="number"{{else}} type="text"{{/if}} name="{{$field.0}}" id="id_{{$field.0}}" value="{{$field.2 nofilter}}"{{if $field.4 eq 'required'}} required{{/if}}{{if $field.5 eq "autofocus"}} autofocus{{elseif $field.5}} {{$field.5 nofilter}}{{/if}} aria-describedby="{{$field.0}}_tip">
```

Comment:
Please use the actual value of `$field.6` for `type` or `text` if it isn't set. This means changing `int` to `number` in the possible values.